.changeset/dull-experts-deny.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-form": minor
```

Comment:
We can include the tokens package to the changeset too to be sure that it gets updated with the new css variables! (related https://github.com/Khan/wonder-blocks/pull/2660#issuecomment-2967880943)